nni/algorithms/compression/pytorch/quantization/quantizers.py

```diff
@@ -475,7 +478,7 @@ def quantize_weight(self, wrapper, **kwargs):
         config = wrapper.config
         module = wrapper.module
         input = kwargs['input_tensor']  # pylint: disable=redefined-builtin
-        weight = copy.deepcopy(wrapper.module.old_weight.data)
+        weight = wrapper.module.weight
```

Comment:
I am not sure, will this cause some backward issue? And other quantizers also use `copy.deepcopy(wrapper.module.old_weight.data)`, should they also be modified?

Reply:
have rebased the master